flang/test/Lower/OpenMP/omp-parallel-allocate-clause.f90

```diff
@@ -0,0 +1,29 @@
+! This test checks lowering of OpenMP parallel Directive with
```

Comment:
Do you know why this test failed previously without the corresponding privates for the allocate variables?

Reply:
Not quite sure as there are semantic checks in place to take care of that.